components/post_markdown/system_message_helpers.jsx

```diff
@@ -243,9 +244,13 @@ function renderPurposeChangeMessage(post) {
         return null;
     }
 
+    const purposeOptions = {
+        channelNamesMap: post.props && post.props.channel_mentions,
+    };
+
     const username = renderUsernameForUserIdAndUsername(post.user_id, post.props.username);
-    const oldPurpose = post.props.old_purpose;
-    const newPurpose = post.props.new_purpose;
+    const oldPurpose = post.props.old_purpose ? renderFormattedText(post.props.old_purpose, purposeOptions) : null;
```

Comment:
Why are we formatting the purpose here? The main place that's used is in the More Channels list where we don't format it

Reply:
I had run across this in actual system messages, and thought that the purpose should be treated similarly to the header. Should the purpose remain plain text instead?

![image](https://user-images.githubusercontent.com/1023171/41788026-333c908c-7618-11e8-852f-b841a5f60d6f.png)

Reply:
Yeah, the purpose is just plain text. We don't format it when it appears in the channel info modal or the more channels list

Reply:
Sounds good -- I've reverted this part of the change.